packages/grant-explorer/src/store.ts

```diff
@@ -77,8 +97,29 @@ export const useCartStorage = create<CartState>()(
           });
         }
       },
-      chainToPayoutToken: ethOnlyPayoutTokens,
-      setPayoutTokenForChain: (chainId: ChainId, payoutToken: VotingToken) => {
+      chainToVotingToken: defaultVotingTokens,
+      getVotingTokenForChain: (chainId: ChainId) => {
+        const tokenFromStore = get().chainToVotingToken[chainId];
+        if (!tokenFromStore) {
+          const defaultToken = getDefaultVotingToken(
+            votingTokensMap[chainId],
+            chainId
+          );
+          console.log(
```

Comment:
Is this debug output or a warning of something fishy going on? Please change to `console.warn` if it's the latter so it has a better chance of standing out.

Reply:
Not really, it's what happens when a new chain is added.